ballerina/vector.bal

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) 2023 WSO2 LLC (http://www.wso2.com) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+enum normType {
```

Comment:
Type names should use CamelCase.
```suggestion
enum NormType {
```

Reply:
Addressed in https://github.com/ballerina-platform/module-ballerina-math.vector/pull/1/commits/b5a3fc20e22f45419415e246e6336ca45b6113c4.